transforms/sort-comp.js

```diff
@@ -119,24 +121,40 @@ const regExpRegExp = /\/(.*)\/([g|y|i|m]*)/;
 
 /**
  * Get indexes of the matching patterns in methods order configuration
- * @param {String} method - Method name.
+ * @param {Object} method
  * @returns {Array} The matching patterns indexes. Return [Infinity] if there is no match.
  */
 function getRefPropIndexes(method) {
+  var methodName = method.key.name;
   var isRegExp;
   var matching;
   var i;
   var j;
   var indexes = [];
-  for (i = 0, j = methodsOrder.length; i < j; i++) {
-    isRegExp = methodsOrder[i].match(regExpRegExp);
-    if (isRegExp) {
-      matching = new RegExp(isRegExp[1], isRegExp[2]).test(method);
-    } else {
-      matching = methodsOrder[i] === method;
+  // Check for static methods
+  if (indexes.length === 0) {
```

Comment:
indexes.length has to equal zero at this point. it just initialized to empty array in the line above.